exchanges/binance/cfutures_types.go

```diff
@@ -206,30 +206,27 @@ type BatchCancelOrderData struct {
 
 // FuturesOrderPlaceData stores futures order data
 type FuturesOrderPlaceData struct {
-	ClientOrderID string  `json:"clientOrderID"`
+	OrderID       int64   `json:"orderId"`
+	Symbol        string  `json:"symbol"`
+	Pair          string  `json:"pair"`
+	Status        string  `json:"status"`
+	ClientOrderID string  `json:"clientOrderId"`
+	Price         float64 `json:"price,string"`
+	AvgPrice      float64 `json:"avgPrice,string"`
+	ExecuteQty    float64 `json:"executeQty,string"`
```

Comment:
Docs [here](https://binance-docs.github.io/apidocs/delivery/en/#new-order-trade) says this field is `executedQty`

Reply:
Nice catch!